src/Microsoft.ML.CodeGen/Templates/Console/Annotation.txt

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
What is this file for?

Reply:
It's auto-gen by Annotation.tt, It could be removed

Reply:
The way we usually handle this is using a `.ttinclude` file. See 

* https://github.com/dotnet/corefxlab/blob/master/src/Microsoft.Data/ColumnArithmeticTemplate.ttinclude
* https://github.com/dotnet/corefx/blob/3b6ec2ba944fb72f921f95b99b6ebf2e7bb3dc5a/src/Common/src/CoreLib/System/Numerics/GenerationConfig.ttinclude

for examples